forms-flow-web/src/components/Form/Item/Edit.js

```diff
@@ -449,377 +249,264 @@ const Edit = React.memo(() => {
           active={formSubmitted}
           spinner
           text={t("Loading...")}
         >
-          <div className="d-flex pb-4 flex-wrap">
-            <div className="col-lg-6 col-md-6 col-sm-6 col-12 px-3">
-              <div>
-                <div id="form-group-title" className="form-group mb-3">
-                  <label htmlFor="title" className="control-label field-required fw-bold mb-3">
-                    {t("Title")}
-                  </label>
-                  <input
-                    data-testid="form-edit-title"
-                    type="text"
-                    className="form-control"
-                    id="title"
-                    placeholder={t("Enter the form title")}
-                    value={form.title || ""}
-                    onChange={(event) => handleChange("title", event)}
-                  />
+          <Errors errors={errors} />
+
+          <Card className="editor-header">
+            <Card.Body>
+              <div className="d-flex justify-content-between align-items-center" style={{ width: "100%" }}>
+                <div className="d-flex align-items-center justify-content-between">
+                  <BackToPrevIcon onClick={backToForm} />
+                  <div className="mx-4 editor-header-text">{form.title}</div>
+                  <span data-testid={`form-status-${form._id}`} className="d-flex align-items-center white-text mx-3">
+                    {processListData.status == "active" ? (
+                      <>
+                        <div className="status-live"></div>
+                      </>
+                    ) : (
+                      <div className="status-draft"></div>
+                    )}
+                    {processListData.status == "active" ? t("Live") : t("Draft")}
+                  </span>
                 </div>
-              </div>
-              <div >
-                <label htmlFor="Description" className="control-label fw-bold mb-2">
-                  {" "}
-                  {t("Description")}
-                </label>
-                <div className="bg-white">
-                  <RichText data-testid="form-edit-description" value={formDescription} onChange={setFormDescription} />
+                <div>
+                  <CustomButton
+                    variant="dark"
+                    size="md"
+                    label={<Translation>{(t) => t("Settings")}</Translation>}
+                    onClick={editorSettings}
+                    dataTestid={"cancelBtndataTestid"}
+                    ariaLabel={"cancelBtnariaLabel"}
+                  />
+                  <CustomButton
+                    variant="dark"
+                    size="md"
+                    className="mx-2"
+                    label={<Translation>{(t) => t("Actions")}</Translation>}
+                    onClick={editorActions}
+                    dataTestid={"cancelBtndataTestid"}
+                    ariaLabel={"cancelBtnariaLabel"}
```

Comment:
- please follow kebab casing while giving datatestid and give relevant naming
- give a meaningful name for aria label
- give translation tag to the arialabeltext (ex : {{(t) => t("Discard Changes")}} )